### ISSUE TITLE
Incorrectly formatted RPC command crashes daemon

### ISSUE THREAD

**Issue by alexbosworth** [lightningnetwork/lnd#117]
Made a mistake when issuing a command and hit a panic (lnd version 0.1.1-alpha - eb73d14a92ad5033099d40579a44128248fc5ded)

```
$ lncli decodepayreq ypyopqokp965ud3wmcsu1f37rp1zmkw5dd5ekos5xtqhximb97wpigeywt1itbb3nnkkme16qpt6r3zcu3yman641u5qkiyhnseoad5gyyyyyyyyyyutbtxh1ory
```

```
panic: runtime error: slice bounds out of range

goroutine 61 [running]:
panic(0x71b880, 0xc420014100)
	/usr/local/Cellar/go/1.7.1/libexec/src/runtime/panic.go:500 +0x1a1
github.com/lightningnetwork/lnd/zpay32.Decode(0x0, 0x0, 0xc4204afa70, 0x4d497, 0x7c4460)
	/Users/Alex/repos/go/src/github.com/lightningnetwork/lnd/zpay32/zbase32check.go:89 +0x1ed
main.(*rpcServer).DecodePayReq(0xc4201ec4e0, 0x1e509c8, 0xc42026a270, 0xc4201900d0, 0x0, 0xc4201d8ad8, 0x2186fc)
```

**Comment by AndrewSamokhvalov** [comment]
@alexbosworth Thanks for creating the issue! 👍  We will try to fix that :)

**Comment by Roasbeef** [comment]
Thanks for the report! This bug has been fixed by d94777acf6fea4a12a0a2a3938bfa98e301f93bf

Closing.
